app/lib/mail/delivery_recorder.rb

```diff
@@ -14,16 +14,16 @@ def initialize(enabled:)
     def delivered_email(mail)
       return unless enabled?
 
-      response = mail.delivery_method.response
+      response = mail.delivery_method.response if mail.delivery_method.respond_to?(:response)
```

Comment:
In what circumstances would this not be expected to respond to `:response`?  Perhaps add a comment to explain?